modules/ROOT/pages/installing-multiple-instances.adoc

```diff
@@ -0,0 +1,226 @@
+. Install Runtime Fabric on the cluster by running the following command:
++
+----
+helm install runtime-fabric rtf/rtf-agent --version <VERSION_FROM_UI> -f values.yaml -n <rtf_namespace>
+----
+
+== Values.yaml File Required Parameters
+The values for these required parameters are set when you create the Runtime Fabric in Runtime Manager. If you’re not using a local registry, use the default values for the registry URL and pull secret. 
+
+[%header%autowidth.spread]
+|===
+| Key | Value | Example
+| `activationData` | Activation Data | YW55cG9pbnQubXVsZXNvZnQuY29tOjBmODdmYzYzLTM3MWUtNDU2Yy1iODg5LTU5NTkyNjYyZjUxZQ==
+| `rtfRegistry` | Registry URL  | US rtf-runtime-registry.kprod.msap.io
+
+EU
+rtf-runtime-registry.kprod-eu.msap.io
+| `pullSecretName` | Registry pull secret | rtf-pull-secret
+| `muleLicense` | Mule license for applications | Mule license key. Must be Base64 encoded.
+|===
+
+[[required-parameters]]
+== Values.yaml Optional Parameters
+Set the following optional parameters as needed: 
+
+[%header%autowidth.spread]
+|===
+| Key | Value | Example
+| `authorizedNamespaces` | Enables shared tenancy | authorizedNamespaces=true
+| `crds.install` | Enables installation of Crds and PriorityClass.  | install=true
+| `crds.keep` | Enables Kubernetes to retain Crds and Priority Class on uninstall. | keep=true
+| `roxy.http.proxy` +
+`proxy.http.no_proxy`
+` | Proxy and no_proxy values | - +http://<user>:<pass>@<10.0.0.1>:<8080>+ +
+- <1.1.1.1:8888,2.2.2.2:9999>
+|`proxy.monitoring.proxy` |Monitoring proxy values | socks5://<user>:<pass>@<10.0.0.2>:<8080>
+|`global.containerLogPaths` | The Filebeat read path | - /var/lib/docker/ +
+ - /var/log/containers +
+ - /var/log/pods
+|===
+
+[NOTE]
+Set the value for `crds.install` and `crds.keep` to `true`, for the first agent being installed on the cluster. +
+Set `crds.install` to `false` for all the subsequent agent installations on the same cluster.
+
+== Values.yaml Reference
+The following is an example of the `values.yml` file:
+
+----
+activationData: <activation data>
+  proxy:
+    http_proxy:
+    http_no_proxy:
+    monitoring_proxy:
+custom_log4j_enabled: true
+muleLicense: <mule license key>
+global:
+  crds:
+    install: true
+    keep: true
+  authorizedNamespaces: false
+  image:
+    rtfRegistry: rtf-runtime-registry.kprod.msap.io
+    pullSecretName: rtf-pull-secret
+  containerLogPaths:
+  - /var/lib/docker/containers
+  - /var/log/containers
+  - /var/log/pods
+----
+
+
+== Deploy Mule Applications
+
+. To deploying Mule applications, create an app-namespace for each of the agent installed: 
++
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: <app-namespace>
+  labels:
+    rtf.mulesoft.com/agentNamespace: <rtf_namespace>
+    rtf.mulesoft.com/envId: <ENVIRONMENT-ID>
+    rtf.mulesoft.com/org: <ORG-ID>
+    rtf.mulesoft.com/role: workers
+----
+[start=2]
+. Create Ingress in new rtf-namespace: +
```

Comment:
Should Ingress be lowercase? Do you need a "the" before Ingress? Any font changes here?